### DISCUSSION TITLE
Account Suspended and Immediately Reinstated – Request for Clarification

### DISCUSSION THREAD

**Discussion by mradarshr** [community/community#140942] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well.

I am Adarh Rathaur (GitHub Username: @ved2007ux, Email: adarshrathaur2007@gmail.com, and I recently faced an unexpected suspension of my GitHub account. While the account has been reinstated, I am unsure about the reasons behind the suspension, as I did not receive any prior warning or notification.

I have reviewed GitHub's terms and policies and believe I have not violated any of them. I would appreciate your help in clarifying the reason for the suspension to ensure compliance with GitHub's policies moving forward.

Additionally, I am encountering an issue with the password recovery process. Despite trying to reset my password, I have not received the reset email. This is causing a delay in accessing my account, which is essential for my daily personal projects and organizational work.
Atleast Allow me to download my data.

The ticket ID for this issue is 3040992. I would be grateful for any assistance you can provide to resolve this matter and prevent future occurrences.

Thank you for your support. @zackfern @martinwoodward @davidstaheli @Bestra @dmilburn

Best regards,
Adarsh Rathaur

**Comment by mradarshr** [comment]
Atleast Allow me to Download all my Repositories Data

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mradarshr, 

Thank you for your patience. The only place to go for answers to account-specific questions like this one is our [Support page](https://support.github.com/contact). The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
